src/Nethermind/Nethermind.State/PersistentStorageProvider.cs

```diff
@@ -240,7 +235,7 @@ protected override void CommitStorageRoots()
         }
 
         // Is overhead of parallel foreach worth it?
-        if (_toUpdateRoots.Count <= 4)
+        if (_toUpdateRoots.Count < 3)
```

Comment:
This magic number is killing me 😢

Reply:
< 3 == ❤️ 

Has to be written out in full 😉